chart/templates/configmap.yaml

```diff
@@ -72,6 +72,85 @@ data:
 
   LOG_SENT_EMAILS: "{{ .Values.email.log_sent_emails }}"
 
+  BACKEND_IMAGE: "{{ .Values.backend_image }}"
+
+  BACKEND_IMAGE_PULL_POLICY: "{{ .Values.backend_pull_policy }}"
+
+
+---
+apiVersion: v1
```

Comment:
I wonder if we should run bg jobs in main namespace to avoid duplicating this configmap, and perhaps other secrets into crawlers namespace.
The crawlers namespace is really supposed to be for crawling..

Reply:
I think I started off using the default namespace but ran into the following error:

```
warning: delete org job could not be started: Error from server (Forbidden): {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"jobs.batch is forbidden: User \"system:serviceaccount:default:default\" cannot create resource \"jobs\" in API group \"batch\" in the namespace \"default\"","reason":"Forbidden","details":{"group":"batch","kind":"jobs"},"code":403}
```

I think we may need to modify the [Service Account permissions](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#service-account-permissions) - reading up on that now.

Reply:
Modified! Replica jobs are kept in crawlers namespace for now to have access to storage secrets, but we can change that if you think it wise.